packages/oc-template-jade/package.json

```diff
@@ -21,6 +21,7 @@
   },
   "license": "MIT",
   "dependencies": {
+    "jade-legacy": "1.11.1",
```

Comment:
I don't think this is actually a dependency. We removed the dependency for using just the checked-in runtime didn't we?

Reply:
The idea was to get a dependencies.io PR so that we are notify for any updates. Perhaps in this case could be a devDependency?